package.json

```
@@ -32,6 +32,7 @@
     "watch": "rollup -cw",
     "dist": "run-s lint types types:strict build build:types docs",
     "postdist": "copyfiles -f bundles/*/dist/* dist && copyfiles -f packages/*/dist/* dist/packages",
+    "codesandbox-ci": "run-s dist prepublish-ci",
```

Would this be more efficient if we remove the doc building, linting and type checking? Is that necessary?

```suggestion
    "codesandbox-ci": "run-s build build:types prepublish-ci",
```